scripts/backup/backup_tool.py

```diff
@@ -0,0 +1,296 @@
+import argparse
```

Comment:
Let's rename to `snapshot.py` and move it to `./scripts`

Reply:
`scripts` folder is for any script.

I think it's better not to move it.